### ISSUE TITLE
🐛 Dropdown is blurry on skill selector component

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#6313]
## 🐛 Bug

I was looking at the skill picker component on My profile and the skill family dropdown seems unreasonably blurry. 
In addition, the contrast with that colour might not be enough. 

Can we look into the blurryness and maybe just leave the text black?

## 🦋 Expected Behaviour

Should not be blurry and it should be readable.

## 🕵️ Details

On my Tablet's built in monitor it actually looks fine but NOT on the monitors attached to my work station.

### TABLET VIEW

![image](https://user-images.githubusercontent.com/16977254/232861204-b094e846-a106-4467-ac2e-3232b4cc335e.png)

### MONITORS VIEW (BLURRY ONE)

![image](https://user-images.githubusercontent.com/16977254/232861333-5fb9b45d-9c47-4a8c-a8a0-9548f019fedf.png)

## 🙋‍♀️ Proposed Solution

Change item colour from `secondary` to `black`.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the bug was addressed.

- [ ] Items appear in a darker colour
- [ ] Items no longer appear blurry at any zoom level

**Comment by petertgiles** [comment]
Hi @Jerryescandon are you sure you to work on this?  The skill picker has no search functionality with the new dashboard flow, right?
![image](https://user-images.githubusercontent.com/8978655/233091915-79270498-a178-4444-a305-d4a1e7d90365.png)
![image](https://user-images.githubusercontent.com/8978655/233092124-520d5f32-2990-4a2c-90b0-8c791352fa35.png)

Do you have any idea how to replicate the blurryness issue?  Is it a browser issue?  Is it OS fractional scaling?  Is it font subpixel scaling?  I'm not sure how devs can address this.

**Comment by Jerryescandon** [comment]
🤔 If we are not using it on the Profile anymore, it might not be so pressing, but we still see this component on the Search page and on the Pool edit page so it's worth taking a look at it anyway.

As to how to reproduce I'm a little lost. The only things I know is that I was on my work Tablet, which I believe has a 2K resolution (it wasnt blurry there) and the blurryness happened only on the external monitors connected through the docking station. The monitor were 1920x1080. @gobyrne also complained about blurryness on a monitor that day. I wonder if it has something to do with the hardware at the new office.

**Comment by petertgiles** [comment]
OK, I'll post a message on devspam and see if anyone has ideas.

**Comment by esizer** [comment]
I think choosing a different colour for the text will help here. As @Jerryescandon said, black text would probably be much better.

**Comment by mnigh** [comment]
Replaced in favour of #6313 with smaller scope.
